### DISCUSSION TITLE
Errors in library

### DISCUSSION THREAD

**Discussion by jumpyjava** [schwabe/ics-openvpn#1741] [Configuration problems]
Hello! No matter how much I try, I can't use this library in any way. When I try to compile a project, errors constantly appear, and in the library itself. It also cannot be imported as a module into Android Studio, since it acts as an application. Maybe I'm doing something wrong? I even tried to import the library into the project and immediately compile it without any action, the result was the same. HELP, I REALLY NEED IT

**Comment by schwabe** [comment]
This is not a library. It is an application. That it acts like an application, is because it is an application.

**Reply by jumpyjava** [reply]
How can I then use this in my application? I need to connect to a VPN server via OpenVpn, everyone advised you
